pandas/io/pytables.py

```diff
@@ -241,20 +242,27 @@ def _tables():
 
 def to_hdf(
     path_or_buf,
-    key,
-    value,
-    mode=None,
+    key: str,
+    value: "NDFrame",
```

Comment:
Does FrameOrSeries not work here? Don’t think will make a difference for typing but from a user / API perspective I think more intuitive than NDFrame

Reply:
updated